src/services/facebookSDK.js

```diff
@@ -1,18 +1,22 @@
-import scriptjs from 'scriptjs';
-
 import { facebookAppId } from 'common/injectGlobals';
 
 const facebookSDK = () =>
-  new Promise((resolve, reject) =>
-    scriptjs(`https://connect.facebook.net/en_US/sdk.js`, () => {
-      global.FB.init({
-        appId: facebookAppId,
-        autoLogAppEvents: true,
-        xfbml: true,
-        version: 'v2.10',
-      });
-      resolve(global.FB);
-    }),
-  );
+  new Promise((resolve, reject) => {
+    const fbapi = global.FB;
+
+    if (!fbapi) {
+      reject({ details: 'Facebook API not defined' });
+      return;
+    }
+
+    global.FB.init({
+      appId: facebookAppId,
+      autoLogAppEvents: true,
+      xfbml: true,
+      version: 'v2.10',
+    });
+
+    resolve(global.FB);
+  });
```

Comment:
what was the problem with the original `scriptjs` approach?

Reply:
No error callback using the script loader

Reply:
I see, might make more sense to have this in `<script>` anyway for now because it's explicitly global, everywhere else, it's referring to `global.FB` :)

Ideally I think we should make this a lazy factory so components can access the FB instance through this rather than reading from `global.FB` but that can happen another time :P